optimize_sidebars.js

```diff
@@ -214,6 +214,10 @@ module.exports = {
 
             {
               "Element templates": [
+                docsLink(
+                  "Element templates",
+                  "components/modeler/desktop-modeler/element-templates/element-templates"
```

Comment:
```suggestion
                  "components/modeler/desktop-modeler/element-templates/"
```

I think this link is also incorrect. The suggestion above resolves correctly when running locally. Note that I think it should also be applied to the other optimize sidebars, too.